### DISCUSSION TITLE
Regarding viewBox and draw.Lines

### DISCUSSION THREAD

**Discussion by Sajal92** [cduck/drawSvg#35] [Q&A] [answered]
Hello everyone,
I wanted to ask a question regarding this library. 

When I passed the height and width in drawSvg.Drawing function as 256, the output svg had it doubled i.e height=512, width=512. Does anybody know what is happening?

Also, when I passed the points in drawSvg.Lines functions the y coordinates of the points are automatically changing to -y in output svg?

Thanks a lot.

Best Regards,
Sajal

**Comment by cduck** [comment, marked as answer]
1. If you post a code snippet that shows your issue, I can help better.  My guess is you used `d.setPixelScale(2)` from example 1 which sets the coordinate system scale to 2x.
2. The inverted y-coordinate was a design decision when the library was used for mathematical drawings.  See #18 and #11 for context.

**Reply by Sajal92** [reply]
Yeah setpixelscale was 2. And aslo I changed the signs of y-coordinates. Now its working. Thanks a lot!
